src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private LocalDate dateFrom;
+    private LocalDate dateTo;
+    private LocalDate currentDate;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder("Report for period ");
```

Comment:
StringBuilder is not a service class. It is just an improved version of String. Let's give it a corresponding name
```suggestion
        StringBuilder result = new StringBuilder("Report for period ");
```